src/main/resources/lib/pwa/manifest-controller.js

```diff
@@ -2,15 +2,14 @@ var portalLib = require('/lib/xp/portal');
 var mustache = require('/lib/mustache');
 
 exports.get = function(req) {
-
-    const startUrl = "./";
+    var startUrl = portalLib.url({ path: "/", type: "absolute" });
```

Comment:
This stopped working at some point. The new (server) uri is: `webapp/com.enonic.app.officeleague/`

Reply:
@reisfmb let's keep it as it was. this is tested and works.